hugo/content/docker-images.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Docker Images"
+draft: false
+description: "Learn how to build and run NGINX Agent docker images"
+weight: 800
+toc: true
+tags: [ "docs" ]
+docs: "DOCS-1230"
+categories: ["configuration"]
+doctypes: ["task"]
+---
+
+## Prerequisites
+1. Docker installation
+1. nginx-repo.crt and nginx-repo.key files (if using NGINX Plus)
```

Comment:
The prerequisites need to link to the appropriate documentation (ours or external) that explains how to complete the step, e.g.
```suggestion
1. Docker must be [installed and running](https://docs.docker.com/engine/install/)
1. (Optional) If you plan to use NGINX Plus, you will need the nginx-repo.crt and nginx-repo.key files. You can download them from [MyF5](https://my.f5.com).
```